### ISSUE TITLE
Unity file conflicts

### ISSUE THREAD

**Issue by simeonradivoev** [simeonradivoev/UniGit#4]
Add support for conflict resolvement for Unity files like prefabs, scenes or materials

- [ ] Scene files
- [ ] Prefabs
- [ ] Materials

**Comment by HeadClot** [comment]
Has this been worked on since it was posted?

**Comment by simeonradivoev** [comment]
Not really. I've mainly got around it using unity's awful [smart merge](https://docs.unity3d.com/Manual/SmartMerge.html).
